stream-chat-android-client/src/main/java/io/getstream/chat/android/client/di/BaseChatModule.kt

```diff
@@ -128,7 +129,7 @@ internal open class BaseChatModule(
     }
 
     // Create Builders from a single client to share threadpools
-    private val baseClient: OkHttpClient by lazy { OkHttpClient() }
+    private val baseClient: OkHttpClient by lazy { customOkHttpClient ?: OkHttpClient() }
```

Comment:
I think the `clientBuilder` method will override the timeouts set on this `OkHttpClient` in all cases

Reply:
Not it only overrides the timeout if the baseClient is not the custom one

Reply:
Neat solution!